### ISSUE TITLE
Failed to load the saved whisper q4_0 model from 0829

### ISSUE THREAD

**Issue by cyita** [intel-analytics/BigDL#8940]
Failed to load the saved whisper q4_0 model from 0829, may be related to the PR https://github.com/intel-analytics/BigDL/pull/8731
```python
from bigdl.llm.transformers import AutoModelForSpeechSeq2Seq


whisper_model_path = "/mnt/disk1/models/whisper-medium"
whisper =  AutoModelForSpeechSeq2Seq.from_pretrained(whisper_model_path, load_in_4bit=True, optimize_model=False)
whisper.config.forced_decoder_ids = None
whisper.save_low_bit("whisper-medium-int4")
whisper =  AutoModelForSpeechSeq2Seq.load_low_bit("whisper-medium-int4", optimize_model=False, trust_remote_code=True,)
```

Error message:
<img width="746" alt="image" src="https://github.com/intel-analytics/BigDL/assets/33650826/1594386b-4496-489c-b301-4c207a6bf6c7">

**Comment by leonardozcm** [comment]
I will take a look at this.

**Comment by leonardozcm** [comment]
After our validation, this may relate to the version of `accelerate`. we need to fix accelerate==0.21.0 for now and see if we can support 0.22.

**Comment by leonardozcm** [comment]
closed as fixed
